sdk/core/azure-core/src/main/java/com/azure/core/http/HttpHeaders.java

```diff
@@ -21,6 +21,25 @@ public class HttpHeaders implements Iterable<HttpHeader> {
     // This map is a case-insensitive key (i.e. lower-cased), but the returned HttpHeader key will be as-provided to us
     private final Map<String, HttpHeader> headers;
 
+    static {
+        HttpHeadersHelper.setAccessor(new HttpHeadersHelper.HttpHeadersAccessor() {
+            @Override
+            public HttpHeaders setNoKeyFormatting(HttpHeaders headers, String formattedName, String name, String value) {
+                return headers.setInternal(formattedName, name, value);
+            }
+
+            @Override
+            public HttpHeader getNoKeyFormatting(HttpHeaders headers, String formattedName) {
+                return headers.getInternal(formattedName);
+            }
+
+            @Override
+            public String getValueNoKeyFormatting(HttpHeaders headers, String formattedName) {
+                return headers.getValueInternal(formattedName);
+            }
+        });
```

Comment:
These methods are still pending a rename, correct?

Reply:
Might has missed that the Helper methods should have `_` in them

Reply:
Reverted as this causes Checkstyle failures